openshift/machine-config-operator
# Bug 2028590: Remove unneeded crio.service.d drop-in

**- What I did**
This drop-in that exists for the baremetal and vsphere platforms is
unnecessary.  Cri-o already respects when $CONTAINER_STREAM_ADDRESS is
set in the environment without having to edit the commandline.

Removing this drop-in reduces code fragility without any change in
functionality.

**- How to verify it**
On a running OCP node, run `crio-status config | grep stream_address` -> It should show the IP address you see in /etc/systemd/system/crio.service.d/20-nodenet.conf and not the default ("127.0.0.1").

**- Description for the changelog**
Removed redundant crio.service.d drop-ins for baremetal and vsphere platforms
